src/lein_monolith/config.clj

```diff
@@ -47,13 +55,17 @@
   are loaded to check for the `:monolith` entry."
   [project]
   (if (:monolith project)
-    project
+    (-> (:root project)
+        (io/file "project.clj")
+        str
+        project/read-raw
+        (add-raw-to-meta project))
```

Comment:
I'm generally :-1: on using threading macros like this. Threading can be really useful when you have a succession of transformations applied to the same kind of data structure, e.g. `->` with `conj`/`assoc`/`dissoc`/`update` etc, or `->>` with `map`/`filter`/other sequence ops. In this case though, every step has a different type than the last, which makes it harder to follow what's going on.

A `let` binding approach would be easier to read and also more compact:
```suggestion
    (let [project-file (io/file (:root project) "project.clj")
          raw-project (project/read-raw (str project-file))]
      (add-raw-to-meta project raw-project))
```